spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -93,10 +93,10 @@ def process(options)
       context "when process terminates" do
         before do
           stub_const("Terminator", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              Process.kill("TERM", 0)
-            end
-          })
+                                   def process(options)
+                                     Process.kill("TERM", 0)
+                                   end
+                                 })
```

Comment:
`end` at 99, 33 is not aligned with `Class.new(DispatchRider::Handlers::Base) {` at 95, 35 or `stub_const("Terminator", Class.new(DispatchRider::Handlers::Base) {` at 95, 10